.yamllint.yml

```diff
@@ -0,0 +1,22 @@
+---
+extends: default
+
+ignore: |
+  internal/configuration/test_resources/config_bad_quoting.yml
+  web/node_modules/
+  compose/
+  config.template.yml
+  internal/configuration/config.template.yml
```

Comment:
These are removed after merging the docs PR. They're temporary to ensure CI passes. 

```suggestion
```

Reply:
ack